creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -538,6 +582,7 @@ const _issueUpdateReplicaSetOp = async (
         })
 
       if (!canReconfig) {
+        response.result = 'skip_update_replica_set'
```

Comment:
might be helpful to get more a bit more granular response from the `_canReconfig()` call as to why it returns false
also `skip_update_replica_set` is not super representative
unfortunately the function is not documented so would be helpful if you could read through and add a summary jsdoc